### PR TITLE
Revert "add Rational64 type (#31gy38v) (#2236)"

### PR DESCRIPTION
**Pull request by benluelo**
This reverts commit 0ff90186311e2e4917752a4c3c84acb1b08076f1, reversing changes made to 85c495df23d0a5a5aacaf07edeced4e0f378cf53.

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-31gy38v XCMP fixes](https://app.clickup.com/t/31gy38v)

File: code/parachain/frame/composable-support/src/math/safe.rs [hunk 1]
```diff
@@ -5,11 +5,9 @@ use sp_runtime::{
 	ArithmeticError,
 };
 
-/// Thin wrapper around [`multiply_by_rational_with_rounding`], transforming any errors into
-/// [`ArithmeticError`] for easier use in pallets and Dispatch-related contexts. See
-/// [`multiply_by_rational_with_rounding`] for more information.
-///
-/// Computes a * b / c, rounding down.
+/// Thin wrapper around [`multiply_by_rational`], transforming any errors into [`ArithmeticError`]
```
Comment (dzmitry-lahoda) [inline code/parachain/frame/composable-support/src/math/safe.rs:8]: was good one

File: code/parachain/frame/composable-support/src/types/const_assertions.rs [hunk 2]
```diff
@@ -1,34 +0,0 @@
...
-///         let _ = AssertNonZero::<N>::OK;
-///         
-///         Self(N)
-///     }
-/// }
-/// ```
-pub struct AssertNonZero<const X: u64>;
-
-impl<const X: u64> AssertNonZero<X> {
-	pub const OK: u8 = 0 - !(X > 0) as u8;
```
Comment (dzmitry-lahoda) [inline code/parachain/frame/composable-support/src/types/const_assertions.rs:33]: was good one

File: code/parachain/frame/composable-support/src/types/rational.rs [hunk 3]
```diff
@@ -1,213 +0,0 @@
...
-			assert_eq!(rational_encoded, u128_representation_encoded);
-		}
-
-		#[test]
-		fn max_encoded_len_same_as_u128() {
-			assert_eq!(u128::max_encoded_len(), Rational64::max_encoded_len())
-		}
-	}
-
-	mod decode {
```
Comment (dzmitry-lahoda) [inline code/parachain/frame/composable-support/src/types/rational.rs:151]: was good

**Review by dzmitry-lahoda — APPROVED** [review]

**Review by vimukthi-git — APPROVED** [review]
LGTM

**Comment by benluelo** [comment]
Resolved all conversations as all the changes are the be reverted.

**Pull request merged**